pkg/plugins/resources/helm/main.go

```diff
@@ -34,19 +34,30 @@ URL:
 	MAJORVERSION = "major"
 	// PATCHVERSION defines patch version identifier
 	PATCHVERSION = "patch"
+	// NOINCREMENT defines if a chart version need to be updated
+	NOINCREMENT = "none"
 )
 
 // Spec defines a specification for an "helmchart" resource
 // parsed from an updatecli manifest file
 type Spec struct {
-	File             string // [target] Define file to update
-	Key              string // [target] Define Key to update
-	Name             string // [source][condition][target] Define Chart name path like "stable/chart"
-	URL              string // [source][condition] Define the chart location
-	Value            string // [target] Define value to set
-	Version          string // [source][condition]
-	VersionIncrement string // [target] Define the rule to incremental the Chart version, accept a list of rules
-	AppVersion       bool   // [target] Boolean that define we must update the App Version
+
+	// Defines the Helm Chart file to update, only for "target"
```

Comment:
I liked your `[target][...] Blabla` notation, any reason not to keep it?

Reply:
I also prefer it but I thought it would less obvious. Let me rollback this.